install.sh

```diff
@@ -153,12 +150,12 @@ echo -e "\n[+] massdns\n"
 
 if [ ! -x "$(command -v massdns)" ]
 then
-	git clone https://github.com/blechschmidt/massdns.git 
-	/tmp/massdns
-	cd /tmp/massdns
+	git clone https://github.com/blechschmidt/massdns.git /var/tmp/massdns
+	cd /var/tmp/massdns
 	make
-	mv bin/massdns /usr/bin/
+	sudo mv bin/massdns /usr/bin/
```

Comment:
I'm avoiding using `sudo` this way because in some environments `sudo` might not be installed.